### ISSUE TITLE
Make "net_device" not required on fortios_vpnipsec_phase1interface

### ISSUE THREAD

**Issue by blkistsg** [fortinetdev/terraform-provider-fortios#101]
net_device should be able to have a default value of "disable" if not defined

```
resource "fortios_vpnipsec_phase1interface" "vpn_hq_wan1"
{ name = "vpn-hq-wan1" interface = "port1" ike_version = "2" peertype = "any" proposal = "aes256-sha256" dhgrp = "21" local_gw = var.local_vpn_eip remote_gw = var.hq_wan1_ip psksecret = "XXXXXXX" nattraversal = "enable" } 

![jikhnug](https://user-images.githubusercontent.com/71094995/96515204-ebe12e00-1232-11eb-9fcf-588dd1e07801.png)

**Comment by frankshen01** [comment]
Hi, @blkistsg You are right! It will be fixed in the next release. Thanks @blkistsg

**Comment by frankshen01** [comment]
Supported and released, please see the latest version(v1.6.7): https://registry.terraform.io/providers/fortinetdev/fortios/latest for details. Thanks @blkistsg .

**Comment by blkistsg** [comment]
Thank you!
